src/commcare_cloud/commands/ansible/service.py

```diff
@@ -274,10 +276,54 @@ class Nginx(AnsibleService):
     inventory_groups = ['proxy']
 
 
-class Elasticsearch(AnsibleService):
+class ElasticsearchClassic(AnsibleService):
+    name = 'elasticsearch-classic'
+    service_name = 'elasticsearch'
+    inventory_groups = ['elasticsearch']
+
+
+class Elasticsearch(ServiceBase):
     name = 'elasticsearch'
+    service_name = 'elasticsearch'
     inventory_groups = ['elasticsearch']
 
+    def execute_action(self, action, host_pattern=None, process_pattern=None):
+        if action == 'status':
+            return ElasticsearchClassic(self.environment, self.ansible_context).execute_action(action, host_pattern, process_pattern)
+        else:
+            if not ask(
+                    "This function does more than stop and start the elasticsearch service. "
+                    "For that, use elasticsearch-classic."
+                    "\nStop will: stop pillows, stop es, and kill -9 if any processes still exist "
+                    "after a period of time. "
+                    "\nStart will start pillows and start elasticsearch "
+                    "\nRestart is a stop followed by a start.\n Continue?", strict=False):
+                return 0  # exit code
+            if action == 'stop' or action == 'restart':
+                self._act_on_pillows(action='stop')
+                self._run_rolling_restart_yml(tags='action_stop')
+
+            if action == 'start' or action == 'restart':
+                self._run_rolling_restart_yml(tags='action_start')
+                self._act_on_pillows(action='start')
+
+    def _act_on_pillows(self, action):
+        # Used to stop or start pillows
+        ansible_context = AnsibleContext(None)
+        service = SERVICES_BY_NAME['pillowtop'](self.environment, ansible_context)
```

Comment:
Any reason not to reference it directly rather than looking it up in the dict?

Reply:
Good idea, https://github.com/dimagi/commcare-cloud/pull/1990/commits/ba1d4017ebb5a0969000fe59355caaa61247a13b